bls/src/test/java/tech/pegasys/teku/bls/BLSPublicKeyTest.java

```diff
@@ -78,6 +78,19 @@ void succeedsWhenComparingInvalidAndValidPublicKeyFails() {
     assertNotEquals(validPublicKey, invalidPublicKey);
   }
 
+  @Test
+  void succeedsWhenInvalidPublicReturnsHashCode() {
```

Comment:
Should probably add another test comparing the invalid key to itself

Reply:
Good idea.